changelog/fragments/autogenerated-makefile-bundle-prerequisite-task.yaml

```diff
@@ -0,0 +1,26 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Add `kustomize` as a prerequiste target of the `bundle`
+      target in the autogenerated Makefile on Golang based
+      operator projects.
```

Comment:
```suggestion
      For Golang based operators, fix an error faced when the [kustomize](https://kubernetes-sigs.github.io/kustomize/) is not installed locally and the Makefile target `bundle` is used by adding the kustomize target on it. 
```

Reply:
Changed. Instead of `when the kustomize is not installed locally...` I've written `when the kustomize binary is not installed locally...`